operatorcourier/identify.py

```diff
@@ -29,7 +29,5 @@ def get_operator_artifact_type(operatorArtifactString):
                 artifact_type = operatorArtifact["kind"]
             if artifact_type is not None:
                 return artifact_type
-
-        msg = 'Courier requires valid CSV, CRD, and Package files'
-        logger.error(msg)
-        raise OpCourierBadArtifact(msg)
+            else:
+                return ""
```

Comment:
What happen when `operatorArtifact` is not a dict?

Now it will return `None`. Should it raise an error instead?

Can we instead of empty string use `None` as representation that there is no operator_artifact?

Reply:
+1, good point